controllers/cards.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-underscore-dangle */
+const Card = require('../models/cards');
+
+module.exports.getCards = (req, res) => {
+  Card.find({})
+    .then((cards) => {
+      res.status(200).send(cards);
+    })
+    .catch(() => {
+      res.status(500).send({ message: 'Ошибка получения карточек' });
+    });
+};
+
+module.exports.postCard = (req, res) => {
+  const { name, link } = req.body;
+  const owner = req.user._id;
+
+  Card.create({ name, link, owner })
+    .then((card) => {
+      res.status(200).send(card);
+    })
+    .catch((err) => {
+      res.status(500).send({ message: err.message });
+    });
+};
+
+module.exports.deleteCard = (req, res) => {
+  const { id } = req.params;
+
+  Card.deleteOne({ _id: id })
+    .then((card) => {
+      if (card.deletedCount !== 0) {
+        return res.status(200).send(card);
+      } else {
+        throw Error()
+      }
```

Comment:
Можно лучше: использовать метод `findByIdAndDelete`.
Метод возвращает запись из базы даных в формате JSON, либо `null`, если карточка не найдена в базе данных.
В этом случае проверять проще:
```suggestion
      if (!card) return res.status(404).send({ 'Карточка не найдена' });
      res.status(200).send(card);
```

Reply:
Исправил